openshift/helper/__init__.py

```diff
@@ -128,7 +168,120 @@ def __lookup_method(self, operation, namespaced):
 
     @staticmethod
     def __get_model(api_version, kind):
+        '''
+        Return the model class for the requested object.
+
+        :param api_version: API version string
+        :param kind: The name of object type (i.e. Service, Route, Container, etc.)
+        :return: class
+        '''
         camel_kind = string_utils.snake_case_to_camel(kind).capitalize()
         model_name = api_version.capitalize() + camel_kind
         model = getattr(client.models, model_name)
         return model
+
+    @property
+    def argspec(self):
```

Comment:
Do we want to abstract the full argspec away into the client library? I was thinking more along the lines of returning the required data to build an argspec, but letting module_utils convert the returned data into an argspec.

I'd worry that just pulling the argspec directly from the library would create a bit too much magic.